docs/content/latest/secure/tls-encryption/tls-authentication.md

```diff
@@ -0,0 +1,231 @@
+---
+title: TLS and authentication
+headerTitle: TLS and authentication
+linkTitle: TLS and authentication
+description: Use authentication in conjunction with TLS encryption.
+headcontent: Use authentication in conjunction with TLS encryption
+image: /images/section_icons/secure/authentication.png
+menu:
+  latest:
+    identifier: tls-authentication
+    parent: tls-encryption
+    weight: 800
+isTocNested: true
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="/latest/secure/authentication/tls-authentication" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL
+    </a>
+  </li>
+</ul>
+
+TLS can be configured in conjunction with authentication using the following configuration flags related to TLS and authentication:
+
+* `ysql_enable_auth` to enable password (md5) authentication
+* `use_client_to_server_encryption` to enable client-server TLS encryption
+* `ysql_hba_conf_csv` to manually set a host-based authentication (HBA) configuration
+
+The default (auto-generated) configuration in the `ysql_hba.conf` file depends on whether auth (`ysql_enable_auth`) and/or TLS (`use_client_to_server_encryption`) are enabled.
+
+The four default cases are shown in the following table.
+
+| | Auth disabled | Auth enabled |
+---|---|---|
+| TLS disabled | host all all all trust</br>(no ssl, no password) | host all all all md5</br>(no ssl, password required) |
+| TLS enabled | hostssl all all all trust</br>(require ssl, no password) | hostssl all all all md5</br>(require ssl and password) |
```

Comment:
nit: the configs here should be formatted as code.
i.e.:  `host all all all trust`, `host all all all md5`, `hostssl all all all trust`, and `hostssl all all all md5` respectively.